src/Components/ExperimentOne/ExperimentOneDisplayInformation.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import BoredAPICaller from "../../ApiTemplates/BoredAPICaller.js";
+import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
+
+const ExperimentOneDisplayInformation = (props) => {
+  const [userName, setUserName] = useState("");
+  const roboHashUrl = "https://robohash.org/" + userName + ".png?size=200x200";
```

Comment:
Template literal here too

Reply:
That's aweomse. I was looking for something like this 😁